.github/workflows/test.yml

```diff
@@ -123,6 +137,12 @@ jobs:
           - {compiler: intel-classic, version: '2021.2'}
           - {compiler: intel-classic, version: '2021.1.2'}
           - {compiler: intel-classic, version: '2021.1'}
+          - {compiler: nvidia-hpc, version: '23.11'}
```

Comment:
@gha3mi  can you update this to all versions before we merge?

Reply:
https://github.com/fortran-lang/setup-fortran/pull/53/commits/7b7f896da30a7ac703543029453d2b0cb520d735 includes all available versions.